package.json

```diff
@@ -123,10 +125,10 @@
     "@angular/language-service": "^16.2.12",
     "@pega/angular-sdk-overrides": "file:pega-angular-sdk-overrides-8.23.10.tgz",
     "@pega/configs": "^0.7.1",
-    "@pega/dx-component-builder-sdk": "~0.1.49",
+    "@pega/dx-component-builder-sdk": "file:pega-dx-component-builder-sdk-23.1.12.tgz",
```

Comment:
Local tgz reference should be removed

Reply:
for now done intentionally, we can generate a tgz file for it.  I can generate one and put in sharepoint, but will only have my 5 template changes to SDK-Angular part.  Torque, can pull the code from bitbucket and generate a tgz file (will have same name) and be able to move forward.  Think for now, like the other tgz files, it is the best, so we don't have to keep updating the package json.

Reply:
dxcb sdk tgz with latest https://pegasystems.sharepoint.com/:u:/s/SP-DXInnovationLab/ESpIl1HyZHtBjslhrf-LbroBrNkrA_L_WcUd2arC9hHegg?e=s3vmXI